src/app/components/dungeon/dungeon.component.ts

```diff
@@ -0,0 +1,44 @@
+import { DungeonService } from '../../services/dungeon-service/dungeon.service';
+import { Component, OnInit } from '@angular/core';
+import { Dungeon } from 'src/app/models/dungeons';
+import { ActivatedRoute } from '@angular/router';
+import { Obstacle } from 'src/app/models/obstacle';
+
+@Component({
+  selector: 'app-dungeon',
+  templateUrl: './dungeon.component.html',
+  styleUrls: ['./dungeon.component.scss']
+})
+export class DungeonComponent implements OnInit {
+
+  private _dungeon: Dungeon;
+  private _heroId: number;
+  private _selectedObstacle: Obstacle;
+  private _events: string[];
+
+  constructor(private dungeonService: DungeonService, private _route: ActivatedRoute) {
+    this._events = [
+      'Rivers and seas boiling.',
+      'Forty years of darkness.',
+      'Earthquakes', 'Volcanoes',
+      'Human sacrifice',
+      'dogs and cats living together – mass hysteria'
+    ];
```

Comment:
I know this is for mock but you should delete the content of the array. At least that was the case for me earlier.